model_api/python/model_api/models/utils.py

```diff
@@ -164,6 +166,7 @@ def __str__(self) -> str:
 class PredictedMask(NamedTuple):
     mask: list[np.ndarray]
     points: list[np.ndarray] | np.ndarray
+    scores: list[float] | np.ndarray
```

Comment:
How about updating `__str__` method to include `scores` information?

Reply:
Added